lib/hooks/useSupabaseAuth.ts

```diff
@@ -7,13 +7,15 @@ const useSupabaseAuth = () => {
   const [user, setUser] = useState<User | null>(null);
   const [sessionToken, setSessionToken] = useState<string | undefined>(undefined);
   const [providerToken, setProviderToken] = useState<string | null | undefined>(undefined);
+  const [userId, setUserId] = useState<string | null | undefined>(undefined);
```

Comment:
```suggestion
  const [userId, setUserId] = useState<string | null | undefined>();
```

Reply:
Good catch, the type hard coding there is also not needed when you don't force set it to undefined values later in the code

Reply:
You're definitely correct